.github/workflows/release.yml

```diff
@@ -0,0 +1,52 @@
+name: Release Images
+
+on:
+  workflow_dispatch:
+    inputs:
+      commit:
+        description: "Commit sha"
+        required: true
+      tag:
+        description: "The tag for the new images"
+        required: true
+
+jobs:
+  retag-and-release:
+    name: "Re-tag and release images"
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+      - name: Set up skopeo
+        run: |
+          skopeo() {
+            # Needed for older docker versions
```

Comment:
have you tried without this? the docker version on gha might be new enough

Reply:
actually wanted to ask you how this was used in the other pipeline